apps/tlon-mobile/src/screens/PostScreen.tsx

```diff
@@ -137,6 +137,7 @@ export default function PostScreen(props: PostScreenProps) {
         contacts={contacts ?? null}
         calmSettings={defaultCalmSettings}
         currentUserId={contactId}
+        parentPost={post!}
```

Comment:
Can we throw an error or something instead of force casting? And/or make it actually optional.

Reply:
We could just return null if we don't have the post, like we do for channel and contactId, will just do that for now I think.